packages/database/src/modelClasses/MapOverlayGroup.js

```diff
@@ -21,8 +21,12 @@ export class MapOverlayGroupModel extends DatabaseModel {
     return MapOverlayGroupType;
   }
 
+  get RootMapOverlayGroupCode() {
+    return ROOT_MAP_OVERLAY_GROUP_CODE;
+  }
+
```

Comment:
I'd use UPPER_SNAKE_CASE for this, since it's a class constant, but we are highly inconsistent on how we expose those so not a big deal